DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -479,9 +479,8 @@ private function normalizeDriverOptions(array $connection): array
      *     mappings:
      *         MyBundle1: ~
      *         MyBundle2: xml
-     *         MyBundle3: { type: annotation, dir: Documents/ }
```

Comment:
Is there a necessity for `type: attribute` to be in examples? Or is it the default?

Reply:
Good catch, I've added the example 👍